tools/cibadmin.c

```diff
@@ -805,31 +785,32 @@ main(int argc, char **argv)
         exit_code = crm_errno2exit(rc);
     }
 
-#if ENABLE_ACL
     if (output != NULL && acl_eval_how != acl_eval_unused) {
         xmlDoc *acl_evaled_doc;
-        rc = pcmk_acl_evaled_as_namespaces(PCMK_ACL_CRED_USER, acl_cred,
-                                           output->doc, &acl_evaled_doc);
+        rc = pcmk__acl_evaled_as_namespaces(acl_cred, output->doc, &acl_evaled_doc);
         if (rc > 0) {
             free_xml(output);
             if (acl_eval_how != acl_eval_ns_full) {
                 xmlChar *rendered = NULL;
-                int rendered_len = 0;
-                enum pcmk__acl_render_how how = (acl_eval_how == acl_eval_ns_simple)
-                                                ? pcmk__acl_render_ns_simple
-                                                : (acl_eval_how == acl_eval_text)
-                                                ? pcmk__acl_render_text
-                                                : (acl_eval_how == acl_eval_color)
-                                                ? pcmk__acl_render_color
-                                                : /*acl_eval_auto*/ isatty(STDOUT_FILENO)
-                                                ? pcmk__acl_render_color
-                                                : pcmk__acl_render_text;
+                enum pcmk__acl_render_how how;
+                if (acl_eval_how == acl_eval_ns_simple) {
+                    how = pcmk__acl_render_ns_simple;
+                } else if (acl_eval_how == acl_eval_text) {
+                    how = pcmk__acl_render_text;
+                } else if (acl_eval_how == acl_eval_color) {
+                    how = pcmk__acl_render_color;
+                } else if (/*acl_eval_auto*/ isatty(STDOUT_FILENO)) {
+                    how = pcmk__acl_render_color;
+                } else {
+                    how = pcmk__acl_render_text;
+                }
```

Comment:
Seeing it expanded, now I'm thinking that a switch would be more readable. The acl_eval_auto/default case would have one if-else